spread.yaml

```
@@ -129,6 +129,10 @@ prepare: |
   install_charmcraft
 
 suites:
+  docs/tutorial/code/:
+    summary: tests tutorial from the docs
+    systems:
+    - ubuntu-22.04-64
```

Can you set this for now

```suggestion
      - ubuntu-22.04-64
    manual: true
```

and I'll follow up with the docs spread tests at https://github.com/canonical/charmcraft/issues/2029